src/backend/marsha/core/api.py

```diff
@@ -48,6 +56,30 @@ def has_permission(self, request, view):
         return False
 
 
+class IsParamsOrganizationAdmin(BasePermission):
```

Comment:
Same here, define persmission in a more appropriated module ? `persmission.py` already exists but maybe to specific ?